### ISSUE TITLE
Bug: Virus

### ISSUE THREAD

**Issue by arozhankov** [quackduck/uniclip#24]
The virustotal service detects the exe file as a virus. Is this a bug?

https://www.virustotal.com/gui/file/b93ad467b78ef00eea3d8e14c7df0c57e1af2722fc96d5f6d2c53f93da7d6901

![Uploading Screenshot 2024-02-27 at 2.27.39 PM.png…]()

**Comment by quackduck** [comment]
Yes, it probably detects all Go binaries as viruses. You can build from source and inspect the code yourself to guarantee that uniclip isn't malware. Here's some more info from the Go devs: https://go.dev/doc/faq#virus
